Python/codecs.c

```diff
@@ -1386,12 +1421,12 @@ PyStatus
 _PyCodec_InitRegistry(PyInterpreterState *interp)
 {
     static struct {
-        const char *name;
+        int policy;
         PyMethodDef def;
-    } methods[] =
+    } error_handlers[] =
```

Comment:
I would prefer to leave this code as it is.

Reply:
Do you mean the name or the fact that I use the static array? In the first case, I'm ok, but in the second I feel that we just have duplicated strings. I'll revert that part tomorrow if you want though! (I just felt that it was nicer to avoid hardcoding the names twice).

Reply:
I don't see how mentioning an error handler by its name such as "strict" is an issue. Your constants add an indirection which makes the code harder to read (what is CODECS_STRICT_ERROR_POLICY? I should check how it's defined).